.cirrus.yml

```diff
@@ -1,8 +1,8 @@
 task:
   freebsd_instance:
     matrix:
-      image: freebsd-12-0-release-amd64
-      image: freebsd-11-2-release-amd64
+      image_family: freebsd-12-1
+      image_family: freebsd-11-3-snap
```

Comment:
11.4-RELEASE was just released, and the 11-3-snap series is probably ended.  Plus, I think it's better to test on official releases than stable snapshots; that's what most of your users will use.  So I suggest:
```yaml
image: freebsd-12-1-release-amd64
image: freebsd-11-4-release-amd64
```

Reply:
An awesome suggestion! Tests are finally green, thanks.